CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# CONTRIBUTING GUIDELINES
+::tada Welcome to Eddie Jaoude community, thank you for your interest in contributing to our projects. :balloon: To get started Please read the [Code of Conduct](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/blob/develop/CODE_OF_CONDUCT.md)
+
+# EDDIE JAOUDE COMMUNITY PROJECTS
+Our projects are open to all contributors, to get started with contributing to our projects, you can check out our projects [here](https://github.com/EddieJaoudeCommunity), and choose a project you wish to make contributions to.
+
+# HOW DO I CONTRIBUTE TO A PROJECT?
+- If this is your first contribution, see [NEWCOMERS.md](). You might also want to take up issues with the [good first issue](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or [help wanted](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/labels/help%20wanted) label.
```

Comment:
I think the link can be relative, something like this...

```suggestion
- If this is your first contribution, see [NEWCOMERS.md](NEWCOMERS.md). You might also want to take up issues with the [good first issue](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or [help wanted](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/labels/help%20wanted) label.
```

I got this from the GitHub Blog https://github.blog/2013-01-31-relative-links-in-markup-files/

Reply:
@eddiejaoude  if this would link to the newcomers.md then its absolutely great, I thought to get the link from github and make a new pull request to add the link.